DiscordChatExporter.Cli.dockerfile

```diff
@@ -40,20 +40,19 @@ LABEL org.opencontainers.image.licenses="MIT"
 
 # Alpine image doesn't come with the ICU libraries pre-installed, so we need to install them manually.
 # We need the full ICU data because we allow the user to specify any locale for formatting purposes.
-RUN apk add --no-cache icu-libs
-RUN apk add --no-cache icu-data-full
+RUN apk add --no-cache icu-libs icu-data-full
 ENV DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false
 ENV LC_ALL=en_US.UTF-8
 ENV LANG=en_US.UTF-8
 
-# Use a non-root user to ensure that the files shared with the host are accessible by the host user
-# https://github.com/Tyrrrz/DiscordChatExporter/issues/851
-RUN adduser --disabled-password --no-create-home dce
-USER dce
+# Use a non-root user
```

Comment:
Can you please add a comment explaining why `1000` was chosen specifically as the ID?

Reply:
You chose 1000 when you did `RUN adduser --disabled-password --no-create-home dce` (it's usually the first user id available). And luckily it happens to map with most installations.

I'm just keeping your chosen id and making it explicit.